NASA-AMMOS/aerie-ui
# Activity presets UI

This resolves #396.

To test preset creation:
1. Navigate to a plan with directives
2. Make changes to the parameter values
3. Click on the "Presets" input
4. Type in a name for the preset
5. Click on the "+" icon
6. Confirm that the changed values are still what you changed them to, but now they have a blue/purple dot next to them
7. Confirm that the selected preset's name is now displayed in the "Presets" input

To test value revert:
1. Change the parameters again
2. Cmd/ctrl click on the orange dots for some of the changed parameters
3. Confirm that the values are reverted to either the preset's values or the default values for the mission model

To test preset switching:
1. Create a couple more presets following the "preset creation" steps
2. Click on the "Presets" input again
3. Select a different preset from the one you currently have selected
4. Confirm that the parameter values are updated to reflect the newly selected preset
